index.html

```diff
@@ -9,18 +9,64 @@
+            <a class="navbar-brand-two mx-auto d-lg-none d-inline-block" href="#">
+                <img src="//placehold.it/40?text=LOGO" alt="logo">
+            </a>
+            <div class="w-100 text-right">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#myNavbar">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+        </div>
+        <div class="collapse navbar-collapse flex-grow-1 text-right" id="myNavbar">
+            <ul class="navbar-nav ms-auto flex-nowrap">
+               
+                <li class="nav-item" >
+                    <a href="#" id="list"  class="nav-link m-2 menu-item text-white">List</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" id="add-nav"  class="nav-link m-2 menu-item text-white">Add a book</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" id="contact-nav" class="nav-link m-2 menu-item text-white">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
     <div class="container">
         <div class="d-flex justify-content-center mb-3 mt-3"><h1>All Awesome Books</h1></div>
-    <div id="book-list"></div>
-    <div class="d-flex justify-content-center mt-1">
-        <form class="d-block">
+        <div id="book-list"></div>
+        <div class="d-flex justify-content-center mt-1" id="add-book">
+        <form id="input-form" class="d-block">
             <input id="title-name" class="form-control" type="text" placeholder="Enter the title" required /><br>
             <input id="author-name" class="form-control" type="text" placeholder="Enter the author" required /><br>
-            <input id="add" type="submit" value="Add" class="btn btn-dark w-100">
+            <button id="add" type="submit" class="btn btn-dark w-100">Add Book</button>
         </form>
     </div>
-   
+    <div id="contact">
+        <h3 class="text-center">Contact information</h3>
+        <p class="text-center">
+            Do you have any questions or you just want to say "Hello"?<br>
+            You can reach out to us!
+        </p>
+        <ul>
+            <li class="text-center">Our e-mail: mail@mail.com</li>
+            <li class="text-center">Our phone number: 0042546998</li>
+            <li class="text-center">Our address: Wasington 30000, Kigali, Rwanda</li>
+        </ul>
     </div>
+    </div>
+<footer class="text-center text-lg-start bg-light text-muted fixed-bottom pt-5">
+
+    <div class="text-center p-4">
+      © 2021 Copyright;
+    </div>
+  </footer>
     <script src="index.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```

Comment:
- [ ] Bootstrap is a nice tool for page styling and responsiveness. But in this project, you're encouraged to use only Javascript and CSS.  Therefore, do not use any library or framework to change the content of the page. Your design do not have to be elaborate: you can simply build something similar to the wireframe below:
![wireframe](https://user-images.githubusercontent.com/26856491/125864227-37a8c79c-6df3-4bf0-a57f-58b78ada7f04.png)

Reply:
Hello thanks for the quick review, about bootstrap, the requirement says that we may choose between CSS or any framework like bootstrap, this is was the reason we used it because we have a limited time. 
![Screenshot from 2021-07-16 00-35-06](https://user-images.githubusercontent.com/48619051/125866459-69fc5485-c014-4180-9cc3-e34b38ffa946.png)

Reply:
Hi @muhenge,

The review guideline clearly states that you're not allowed to use any framework. I also have the same in the project requirements too here:
![one](https://user-images.githubusercontent.com/26856491/125868222-9d2245f6-8d8a-4a42-9d3f-4c544b894327.png)

Reply:
The guideline on style clearly states that you can use bootstrap to style the content.
![Screenshot 2021-07-15 201501](https://user-images.githubusercontent.com/86670011/125869519-b665053a-f096-4eb3-8c1c-c194cd89c909.png)

Reply:
Sorry guys, it was just a little mixed 🙏 , thank you for your understanding.

Reply:
It's alright man, don't worry 🙏. Yes, the guidelines were a bit ambiguous and easy to misunderstand.